src/input/source/led/multicolor_chassis.rs

```diff
@@ -0,0 +1,167 @@
+use crate::{
+    config::LedFixedColor,
+    input::source::{SourceInputDevice, SourceOutputDevice},
+    udev::device::UdevDevice,
+};
+use std::{error::Error, fmt::Debug, path::PathBuf};
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum MultiColorChassisError {
+    #[error("Error reading multi_index: {0}")]
+    MultiIndexError(std::io::Error),
+
+    #[error("File multi_intensity not found: {0}")]
+    MultiIntensityNotFound(PathBuf),
+
+    #[error("Error updating multi_intensity: {0}")]
+    MultiIntensityUpdateError(std::io::Error),
+
+    #[error("Unsupported index type: {0}")]
+    UnsupportedIndexType(String),
+}
+
+enum IndexType {
+    RGB,
+}
```

Comment:
To make clippy happy:

```suggestion
enum IndexType {
    Rgb,
}
```

Reply:
Oh no my eyes! Noooo not this.

Reply:
We can also do an #allow here to make clippy ignore it since it's an acronym